src/popup/popup.js

```
@@ -29,57 +29,103 @@ let bmFileType
+// Interfaces - SOAP - https://devmcnichols.bigmachines.com/rest/v1/interfaceCatalogs/soapCatalog/services/Security_v1
 
 // Stylesheets - Stylesheet Manager - https://devmcnichols.bigmachines.com/admin/ui/branding/edit_site_branding.jsp
 // Stylesheets - Header & Footer = https://devmcnichols.bigmachines.com/admin/ui/branding/edit_site_branding.jsp
 
-// Documents  -global xsl - https://devmcnichols.bigmachines.com/admin/document-designer/4653759/editor/134737862
+// Documents - global xsl - https://devmcnichols.bigmachines.com/admin/document-designer/4653759/editor/134737862
 
 
 // URL matchers for different sections and rule types
-const URL_MATCHERS = {
+let URL_MATCHERS = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'URL_MATCHERS' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)